frb_codegen/src/generator/dart/ty_delegate.rs

```diff
@@ -60,6 +73,27 @@ impl TypeDartGeneratorTrait for TypeDelegateGenerator<'_> {
             IrTypeDelegate::PrimitiveEnum { ir, .. } => {
                 format!("return {}.values[raw];", ir.dart_api_type())
             }
+            #[cfg(feature = "chrono")]
+            IrTypeDelegate::Time(ir) => match ir {
+                IrTypeTime::Local => "
+                if (kIsWeb) {
+                  return DateTime.fromMillisecondsSinceEpoch(_wire2api_i64(raw), isUtc: false);
+                }
+                return DateTime.fromMicrosecondsSinceEpoch(_wire2api_i64(raw), isUtc: false);"
+                    .to_owned(),
+                IrTypeTime::Naive | IrTypeTime::Utc => "
+                if (kIsWeb) {
+                  return DateTime.fromMillisecondsSinceEpoch(_wire2api_i64(raw), isUtc: true);
+                }
+                return DateTime.fromMicrosecondsSinceEpoch(_wire2api_i64(raw), isUtc: true);"
```

Comment:
It is possible we combine the two literal strings, since only diff is "isUtc"

Reply:
resolved in [7b122ab](https://github.com/fzyzcjy/flutter_rust_bridge/pull/694/commits/7b122ab6d969af3e5a6e4c8e05051c3a7c25600a)